src/main/scala/xebia/cakemix/AskTimeoutProvider.scala

```diff
@@ -17,7 +17,9 @@ trait AskTimeoutProvider {
 }
 
 object AskTimeoutProvider {
-  def fromConfig(config: Config, key: String): Timeout = {
+  private val AskTimeoutConfigKey = "cakemix.ask-timeout"
```

Comment:
IMHO it is always good practice for Scala libs that depend on one or more (typesafe) config values to come with a reference.conf file that contains a valid default. Otherwise code will fail at runtime with unbound config values.